certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/user/model/SrsActive.java

```diff
@@ -0,0 +1,5 @@
+package se.inera.intyg.certificateservice.domain.user.model;
+
+public record SrsActive(boolean value) {
+
```

Comment:
As we discussed, maybe we should set value to false if SrsActive is created with null. It could happen if a consumer for whatever reason don't provide a value for SrsActive when calling CS.